### PR TITLE
updated README.md and added spec.md

### PR DESCRIPTION
**Pull request by somersbmatthews**

### CODE REVIEW

**Comment by somersbmatthews** [comment]
#14

**Comment by wlan0** [comment]
thanks @somersbmatthews

File: README.md [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
 # Container Object Storage Specification
 
 Kubernetes specific Container Object Storage Interface (COSI) components.
 
 ## Community, discussion, contribution, and support
 
+If you are new to a SIG Storage project, check out this [contributing guide](https://github.com/kubernetes/community/blob/master/sig-storage/CONTRIBUTING.md) 
+and the [community page](https://github.com/kubernetes/community/tree/master/sig-storage). 
+
+If you are new to the SIG Storage COSI project, check out the [spec](https://github.com/kubernetes-sigs/container-object-storage-interface-spec/blob/master/spec.md), [KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1979-object-storage-support), and [project board](https://github.com/orgs/kubernetes-sigs/projects/8#card-40949963).
```
[the remark this answers is not in the document]
Comment (somersbmatthews) [reply README.md]: updated all readme's

[the remark this answers is not in the document]
Comment (somersbmatthews) [reply README.md]: done

File: README.md [hunk 2]
```diff
@@ -1,18 +1,40 @@
...
+COSI repos:
+
+|Repo | Images | 
+|----------------|-----------|
+| [Spec](https://github.com/kubernetes-sigs/container-object-storage-interface-spec) | N/A  |
+| [API](https://github.com/kubernetes-sigs/container-object-storage-interface-api) | N/A |
+| [Controller](https://github.com/kubernetes-sigs/container-object-storage-interface-controller) | [cosi-controller](https://quay.io/repository/containerobjectstorage/objectstorage-controller?tab=tags) |
+| [Provisioner Sidecar](https://github.com/kubernetes-sigs/container-object-storage-interface-provisioner-sidecar) | cosi-provisioner TODO |
+| [CSI Adapter](https://github.com/kubernetes-sigs/container-object-storage-interface-csi-adapter) | cosi-node-adapter TODO |
+\
```
Comment (sajanjswl) [inline README.md:10]: Please fix this.

Comment (somersbmatthews) [reply README.md:10]: There are no images created on quay.io to my knowledge for the provisioner-sidecar and the CSI adapter. Were you referring to something else?

File: README.md [hunk 3]
```diff
@@ -1,18 +1,40 @@
...
+COSI deprecated repos:
+
+[Spec](https://github.com/container-object-storage-interface/spec) \
+[API](https://github.com/container-object-storage-interface/api) \
+[Manager (i.e. Controller)](https://github.com/container-object-storage-interface/cosi-controller-manager) \
+[Provisioner Sidecar](https://github.com/container-object-storage-interface/cosi-provisioner-sidecar) \
+[Ephemeral CSI Driver](https://github.com/container-object-storage-interface/ephemeral-csi-driver) 
+
```
Comment (xing-yang) [inline README.md:18]: The table does not look right:

https://github.com/kubernetes-sigs/container-object-storage-interface-spec/blob/63d3ccc27e80c3eb2db3b09520d7665716cff5f7/README.md

File: README.md (hunk 2 above)
Comment (somersbmatthews) [reply README.md:10]: oh I see, will fix

**Comment by somersbmatthews** [comment]
@sajanjswl @xing-yang squashed commits and removed table.

**Review by sajanjswl — APPROVED** [review]

**Pull request merged**